<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>suggest_user_event</class>
 <widget class="QWidget" name="suggest_user_event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Suggestions</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);
font: 12pt &quot;Courier&quot;;</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QTimeEdit" name="begin_time_edit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="duration_input">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="end_time">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Courier&quot;;</string>
      </property>
      <property name="text">
       <string>Duration:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="begin_time">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Courier&quot;;</string>
      </property>
      <property name="text">
       <string>Deadline Time:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="date_text">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Courier&quot;;</string>
      </property>
      <property name="text">
       <string>Deadline Date:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Courier&quot;;
background-color: rgb(255, 0, 128);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Courier&quot;;
background-color: rgb(227, 201, 69);</string>
   </property>
   <property name="text">
    <string>Select Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="attendees">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Courier&quot;;</string>
   </property>
   <property name="text">
    <string>Suggested Times</string>
   </property>
  </widget>
  <widget class="QListWidget" name="time_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="suggest_times">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Courier&quot;;
background-color: rgb(227, 201, 69);</string>
   </property>
   <property name="text">
    <string>Suggest Times</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
